tests/apply/fromfile.c

```diff
@@ -43,7 +43,9 @@ static int apply_patchfile(
 
 	if (error == 0) {
 		cl_assert_equal_i(new_len, result.size);
-		cl_assert(memcmp(new, result.ptr, new_len) == 0);
+		if (new) {
```

Comment:
```suggestion
		if (new_len)
```

Reply:
Hmm, why check the `*_len` parameter instead of the variable itself?